bip-0039.mediawiki

```diff
@@ -165,6 +165,7 @@ Rust:
 Swift:
 * https://github.com/CikeQiu/CKMnemonic
 * https://github.com/yuzushioh/WalletKit
```

Comment:
(Would like to mention, that other links doesn't direect to the class itself)

Reply:
They include the example of how generate a mnemonic right on top of their readme.